packages/tools/src/utilities/rtstruct/getRTROIObservationsSequence.js

```diff
@@ -0,0 +1,8 @@
+export default function getRTROIObservationsSequence(toolData, index) {
+  return {
+    ObservationNumber: index + 1,
+    ReferencedROINumber: index + 1,
+    RTROIInterpretedType: 'Todo: type',
+    ROIInterpreter: 'Todo: interpreter',
+  };
+}
```

Comment:
We don't need the RT stuff in tools, rather just the labelmap to contour. RT can remain in the adapter, and in adapters we only call the the converters and build the RT scheme

Reply:
Some of the stuff needs to be in tools, or is properly in tools because it is a tool, so leaving htose ones.